Resources/Prototypes/_StarLight/Entities/Objects/Specific/Abductor/vending_machine_restock.yml

```diff
@@ -0,0 +1,15 @@
+- type: entity
+  parent: BaseVendingMachineRestock
+  id: VendingMachineRestockAbductorDispenser
+  name: alien dispenser restock
+  description: A box filled with weird organs. Load it into a alien dispenser to begin.
+  components:
+    - type: VendingMachineRestock
+      canRestock:
+        - AbductorGlandInventory
+    - type: Sprite
```

Comment:
No sprite rsi path defined?

Reply:
since it parents from `BaseVendingMachineRestock` so just like all the other vending machine restocks the sprite RSI path is defined in `Resources/Prototypes/Entities/Objects/Specific/Service/vending_machine_restock.yml` and the RSI used is `Objects/Specific/Service/vending_machine_restock.rsi`